pallets/parachain-staking/src/tests.rs

```diff
@@ -96,45 +96,92 @@ fn online_offline_works() {
 }
 
 #[test]
-fn join_collator_candidates() {
+fn can_join_candidates_with_valid_bond() {
 	ExtBuilder::default()
-		.with_balances(vec![
-			(1, 1000),
-			(2, 300),
-			(3, 100),
-			(4, 100),
-			(5, 100),
-			(6, 100),
-			(7, 100),
-			(8, 9),
-			(9, 4),
-		])
-		.with_collators(vec![(1, 500), (2, 200)])
-		.with_nominations(vec![(3, 1, 100), (4, 1, 100), (5, 2, 100), (6, 2, 100)])
+		.with_balances(vec![(1, 1000)])
+		.build()
+		.execute_with(|| {
+			assert_ok!(Stake::join_candidates(Origin::signed(1), 10u128,));
+		});
+}
+
+#[test]
+fn cannot_join_candidates_twice() {
```

Comment:
```suggestion
fn cannot_join_candidates_if_candidate() {
```

or `if_already_candidate`